src/pages/saksbehandling/steg/sats/Sats.tsx

```diff
@@ -77,25 +77,30 @@ const utledSats = (values: FormData, harEPS: boolean, epsAlder?: Nullable<number
     return null;
 };
 
-const schema = yup.object<FormData>({
-    delerSøkerBolig: yup.boolean().defined().when('epsFnr', {
-        is: null,
-        then: yup.boolean().required(),
-    }),
-    mottarEktemakeEllerSamboerSU: yup
-        .boolean()
-        .defined()
-        .test('eps mottar SU', 'Feltet må fylles ut', function (mottarSu) {
-            const { epsFnr, epsAlder } = this.parent;
```

Comment:
epsFnr var fjernet en god tid tilbake. epsAlder var også fjernet for ikke så lenge siden.